test_config.py fail on a machine with access to /lustre/storeA and /lustre/storeB

``` console
$ python -m pytest tests/test_config.py::test_Config_read_config
Test session starts (platform: linux, Python 3.10.12, pytest 8.1.1, pytest-sugar 1.0.0)
rootdir: /home/alvarov/Python/pyaerocom
configfile: pyproject.toml
plugins: xdist-3.5.0, dependency-0.6.0, cov-4.1.0, sugar-1.0.0
...
tests/test_config.py:134: AssertionError
----------------------------------------------------- Captured log call ------------------------------------------------------
INFO     pyaerocom.config:config.py:275 Checking access to: /home/alvarov
INFO     pyaerocom.config:config.py:275 Checking access to: /lustre/storeB/project
INFO     pyaerocom.config:config.py:275 Checking access to: /home/alvarov/aerocom/aerocom1
INFO     pyaerocom.config:config.py:275 Checking access to: /lustre/storeB/project/aerocom/aerocom1
INFO     pyaerocom.config:config.py:275 Checking access to: /home/alvarov/aerocom
INFO     pyaerocom.config:config.py:275 Checking access to: /lustre/storeB/project/aerocom

 tests/test_config.py ⨯✓✓✓✓✓✓✓✓✓✓s✓✓✓s                                                                         100% ██████████
================================================== short test summary info ===================================================
FAILED tests/test_config.py::test_Config_read_config[metno-None-False-False-0] - AssertionError
FAILED tests/test_config.py::test_Config_read_config[metno-None-True-False-0] - AssertionError
FAILED tests/test_config.py::test_Config_read_config[metno-None-True-True-0] - AssertionError
FAILED tests/test_config.py::test_Config_read_config[metno-/home/alvarov-True-True-2] - AssertionError

Results (0.12s):
      13 passed
       4 failed
         - tests/test_config.py:115 test_Config_read_config[metno-None-False-False-0]
         - tests/test_config.py:115 test_Config_read_config[metno-None-True-False-0]
         - tests/test_config.py:115 test_Config_read_config[metno-None-True-True-0]
         - tests/test_config.py:115 test_Config_read_config[metno-/home/alvarov-True-True-2]
       2 skipped
```

_Originally posted by @avaldebe in https://github.com/metno/pyaerocom/issues/1037#issuecomment-1999805178_

In all the failed tests len(cfg.DATA_SEARCH_DIRS) is 30. The tests expect 0, 0, 0 and 2 respectively.

--- a/tests/test_config.py
+++ b/tests/test_config.py
@@ -113,26 +113,24 @@ def test_Config_has_access_users_database():
 
 
 @pytest.mark.parametrize(
-    "cfg_id,basedir,init_obslocs_ungridded,init_data_search_dirs,data_searchdirno",
+    "cfg_id,basedir,init_obslocs_ungridded,init_data_search_dirs",
     [
-        ("metno", None, False, False, 0),
-        ("metno", None, True, False, 0),
-        ("metno", None, True, True, 0),
-        ("metno", f"/home/{USER}", True, True, 2),
-        ("users-db", None, False, False, 0),
+        ("metno", None, False, False),
+        ("metno", None, True, False),
+        ("metno", None, True, True),
+        ("metno", f"/home/{USER}", True, True),
+        ("users-db", None, False, False),
     ],
 )
-def test_Config_read_config(
-    cfg_id, basedir, init_obslocs_ungridded, init_data_search_dirs, data_searchdirno
-):
+def test_Config_read_config(cfg_id, basedir, init_obslocs_ungridded, init_data_search_dirs):
     cfg = testmod.Config(try_infer_environment=False)
     cfg_file = cfg._config_files[cfg_id]
     assert Path(cfg_file).exists()
     cfg.read_config(cfg_file, basedir, init_obslocs_ungridded, init_data_search_dirs)
     if not cfg.has_access_lustre:
         pytest.skip(f"Skipping since {cfg._LUSTRE_CHECK_PATH} directory not accessible")
-    assert len(cfg.DATA_SEARCH_DIRS) == data_searchdirno
-    assert len(cfg.OBSLOCS_UNGRIDDED) == 0
+    assert all([Path(idir).exists() for idir in cfg.DATA_SEARCH_DIRS])
+    assert cfg.OBSLOCS_UNGRIDDED
     assert Path(cfg.OUTPUTDIR).exists()
     assert Path(cfg.COLOCATEDDATADIR).exists()
     assert Path(cfg.CACHEDIR).exists()

